src/components/primitives/Box/index.tsx

```diff
@@ -52,6 +52,7 @@ const Box = ({ children, ...props }: IBoxProps, ref: any) => {
           y: lgrad.end[1],
         };
       }
+      if ('bg' in safeAreaProps) delete safeAreaProps['bg'];
```

Comment:
@cyrus25 we need to delete background, bgColor and backgroundColor as well. Please check line 23.